Content.Client/SS220/MindExtension/UI/GhostBodyCard.xaml.cs

```diff
@@ -0,0 +1,70 @@
+using Content.Client.Administration.UI.CustomControls;
+using Content.Shared.SS220.MindExtension;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.SS220.MindExtension.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class GhostBodyCard : UIWidget
+{
+    public Action<NetEntity>? FollowBodyAction;
+    public Action<NetEntity>? ToBodyAction;
+
+    private TrailPoint _body;
+    public GhostBodyCard()
+    {
+        RobustXamlLoader.Load(this);
+    }
+    public void Init(TrailPoint body)
+    {
+        _body = body;
+
+        BodyNameLabel.Text = body.MetaData.EntityName;
+        BodyDescriptionLabel.Text = body.MetaData.EntityDescription;
+
+        BodyStateLabel.FontColorOverride = Color.Red;
+
+        switch (body.State)
+        {
+            case BodyStateToEnter.Avaible:
+                FollowButton.Disabled = false;
+                ToBodyButton.Disabled = false;
+
+                BodyStateLabel.FontColorOverride = Color.Green;
+                BodyStateLabel.Text = "(Доступен)";
+                break;
+            case BodyStateToEnter.Abandoned:
+                FollowButton.Disabled = false;
+                BodyStateLabel.Text = "(Безвозвратно вами покинут)";
```

Comment:
```suggestion
                BodyStateLabel.Text = Loc.GetString("body-state-to-enter-abandoned-text");
```
Типо вот таких и везде где у тебя буквы вместо LocId